libcxx/include/__bit_reference

```diff
@@ -290,6 +290,15 @@ _LIBCPP_CONSTEXPR_SINCE_CXX20 _LIBCPP_HIDE_FROM_ABI __bit_iterator<_Cp, false> _
 template <class _Cp, bool _IsConst>
```

Comment:
Not attached: Please add a short description of *what* the optimization is in the PR description.